blogposts/inside-rust/01-announcement.md

```diff
@@ -86,6 +86,8 @@ There are two common types of forced unwinding:
   * It may, of course, cause programs to abort that used to execute
     successfully. This could occur if a panic would've been caught and
     recovered.
+* Changing the ABI (the `"C"` in `extern "C"`) of functions in the libc crate is
+  a breaking change: function pointers of different ABIs have different types.
```

Comment:
We consider this to be an unacceptable cost, correct? (This should probably be explicit either way.)

Reply:
I think so. Unless we want to play around with fancy coercion rules... (but I'd rather not)

Reply:
Yeah, then let's just say something like:

```suggestion
  * **We cannot change** the ABI (the `"C"` in `extern "C"`) of functions in the libc crate, **because this would be** a breaking change: function pointers of different ABIs have different types.
```